cake/BuildExternals.cake

```diff
@@ -713,28 +713,22 @@ Task ("externals-angle-uwp")
 });
 
 Task ("externals-harfbuzz")
-    .WithCriteria (
-        !FileExists (HARFBUZZ_PATH.CombineWithFilePath ("harfbuzz/README")) ||
-        !FileExists (HARFBUZZ_PATH.CombineWithFilePath ($"harfbuzz-{GetVersion ("harfbuzz", "release")}.tar.bz2")))
     .Does (() =>
 {
-    var version = GetVersion ("harfbuzz", "release");
-    var url = $"https://github.com/behdad/harfbuzz/releases/download/{version}/harfbuzz-{version}.tar.bz2";
-    DirectoryPath root = HARFBUZZ_PATH;
-    FilePath archive = root.CombineWithFilePath ($"harfbuzz-{version}.tar.bz2");
-
-    EnsureDirectoryExists (root);
-    CleanDirectory (root);
-
-    DownloadFile (url, archive);
-    DecompressArchive (archive, root);
-    MoveDirectory (root.Combine ($"harfbuzz-{version}"), HARFBUZZ_PATH.Combine ("harfbuzz"));
-
     if (!IsRunningOnWindows ()) {
-        RunProcess ("bash", new ProcessSettings {
-            Arguments = "configure",
-            WorkingDirectory = HARFBUZZ_PATH.Combine ("harfbuzz"),
-        });
+        if (!FileExists (HARFBUZZ_PATH.CombineWithFilePath ("configure"))) {
```

Comment:
This part is causing the issues. Will remove these steps because we don't need them if we create a config.h in advance like we already do with other platforms. This will make some dependencies on the build machine redundant so they can safely be removed.

Reply:
That will be much better. If we are trying to do womething, but it is missing from the platform then the build should fail, and not try and find an alternative or something.